cordapp-cpk2/README.md

```diff
@@ -3,7 +3,7 @@
 
 ## Purpose.
 Applying this plugin to a project declares that the project should create a CPK-format CorDapp. The CPK-format
-CorDapp is a ZIP file with a `.cpk` extension that contains the output of the `jar` task (the "main" jar), along
+CorDapp is a ZIP file with a `.jar` extension that contains the output of the `jar` task (the "main" jar), along
```

Comment:
Might be worth clarifying how you identify the "main" JAR vs the "CorDapp" JAR given they now both end with JAR. Generally I see that there are two JARs built - one with a `-sources` suffix and one without.

Reply:
These two terms refer to the same jar file in version 2. The `-sources` file is the output of `sourcesJar` which is outside the scope of this document.

Reply:
Then isn't the text confusing? I still read it as there were two separate JAR files.

Reply:
I'll remove talk of the main jar.

Reply:
Done